src/error.rs

```diff
@@ -17,15 +18,70 @@ pub enum Rav1dError {
     /// which is more optimal since `0` is no error for [`Dav1dResult`].
     EGeneric = 1,
 
-    ENOENT = libc::ENOENT as u8,
-    EIO = libc::EIO as u8,
-    EAGAIN = libc::EAGAIN as u8,
-    ENOMEM = libc::ENOMEM as u8,
-    EINVAL = libc::EINVAL as u8,
-    ERANGE = libc::ERANGE as u8,
-    ENOPROTOOPT = libc::ENOPROTOOPT as u8,
+    /// Invalid buffer.
+    ///
+    /// No Sequence Header OBUs were found in the buffer.
+    InvalidBuffer = libc::ENOENT as u8,
```

Comment:
```suggestion
    /// No entity.
    ///
    /// No Sequence Header OBUs were found in the frame.
    NoEntity = libc::ENOENT as u8,
```

I think `NoEntity` is more appropriate for `ENOENT` given that it's much closer to `ENOENT`s meaning (no entity/entry) and fits well with its usage here, no seq hdr OBU.

Reply:
The only place its used is `rav1d_parse_sequence_header` and the C version of that function has the "no sequence header..." description for that error, so I'd agree something like  NoSequenceHeader makes sense

Reply:
Hmm, possibly, although dav1d may add another usage in the future and we do want to keep a one to one mapping. Not sure though.